string object
float32 go_x
float32 go_y
float32 until_x
float32 until_y
---
int8 status
float32 time
